src/mame/elektor/avrmax.cpp

```diff
@@ -52,8 +52,8 @@ For the German magazine, AVR-Max-Schachzwerg and CC2-Schachzwerg are in 06/2009.
 #include "screen.h"
 
 // internal artwork
-#include "avrmax.lh"
-#include "atm18mcc.lh"
+#include "avrmax.lh" // clickable
+#include "atm18mcc.lh" // clickable
 
```

Comment:
It looks like you modified an older version of this file and copied it over? See the readded 'clickable' comment, and also the slightly modified comment at the top of the driver.